File: Lista3/Lista3_11800910/exec4/CD.java
/**
 * Classe CD
 */
public class CD extends Estoque {
    
    /**
     * expande a classe Estoque com as info do CD
     * @param nome
     * @param num
     * @param qte
     */
    public CD(String nome, long num, int qte){
        super(nome, num, qte);
    }

    @Override
    public String toString(){
        String s = "Tipo: CD\n";
        s += "Nome: " + getNome() + "\n";
        s += "Código de Barras: " + getNum() + "\n";
        s += "Quantia em Estoque: " + getQte() + "\n";
        return s;
    }

}

File: Lista4/exec4/CD.py
from estoque import *

class CD(estoque):

    #  * expande a classe Estoque com as info do CD
    #  * @param nome
    #  * @param num
    #  * @param qte
    def __init__(self, nome: str, num: int, qte: int):
        super().__init__(nome, num, qte)

    def __str__(self) -> str:
        s = '\nTipo: CD\n'
        s += "Nome: " + str(self.getNome()) + "\n"
        s += "Código de Barras: " + str(self.getNum()) + "\n"
        s += "Quantia em Estoque: " + str(self.getQte()) + "\n"
        return s
